src/library/coverartutils.cpp

```diff
@@ -56,6 +56,18 @@ QImage CoverArtUtils::extractEmbeddedCover(
     return image;
 }
 
+QImage CoverArtUtils::extractEmbeddedCover(
+        TrackPointer pTrack) {
+    QImage image;
+    // Both resetMissingTagMetadata = false/true have the same effect
+    constexpr auto resetMissingTagMetadata = false;
```

Comment:
that sounds like a bug?

Reply:
I did not investigate that. The comment is a literal copy form the previous version.

Reply:
If you don't read any other metadata by passing `nullptr` the option has no effect.